Cargo.toml

```diff
@@ -12,8 +12,16 @@ libarchive = "3.0"
 jsonglib = { name = "json-glib-1.0", version = "1" }
 polkitgobject = { name = "polkit-gobject-1", version = "0" }
 rpm = "4"
+# libdnf dependencies
 librepo = "1"
 libsolv = "0.7"
+libsolvext = "0.7"
+gpgme = "1"
+openssl = "1"
+libcurl = "7"
+sqlite3 = "3"
+modulemd = { name = "modulemd-2.0", version = "2" }
+jsonc = { name = "json-c", version = "0" }
```

Comment:
Oh man, hadn't realized we linked in two JSON libraries (and I guess with serde, that's three!).

Reply:
Yeah I too was surprised while working on this, then I ran `ldd /usr/bin/rpm-ostree` and realized just how much *stuff* is linked into our address space...